python/cuml/benchmark/algorithms.py

```diff
@@ -112,9 +121,15 @@ def run_cuml(self, data, **override_args):
         if self.data_prep_hook:
             data = self.data_prep_hook(data)
         if self.accepts_labels:
-            cuml_obj.fit(data[0], data[1])
+            if self.bench_func is not None:
```

Comment:
Same as above

Reply:
Actually, I was hoping to make these functions part of the public API- especially the `fit_transform`, since this is a common function in sklearn's `Transformer` API. Given the idea above, which would always use a `bench_func`, what do you think about making a separate file for the different funcs we use? Maybe something like `bench_funcs.py`?